_component_design/switches.md

```diff
@@ -1,85 +1,105 @@
 ---
 title: Switches
-parent: null
+parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  A switch is used for a binary decision; and only for the purpose of yes/no or on/off. They are especially useful when building mobile based designs.
-preview-image: components/preview-selectors.svg
+  A switch allows a user to commit a binary decision and is used exclusively or yes or no, and on or off decisions.
+  They are especially useful when building mobile based designs.
+preview-image: preview-images/switches.svg
 resource: true
 need: selectors
 status: in-progress
-last-modified: 2017-08-17
+last-modified: 2017-10-23
 ---
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation, and improve
-keyboard accessibility." %}-->
 
-The switch should only be used in **mobile** designs. Enabling preferences or
-settings is a prime use case for this component. The switch can be used with or
-without text, depending on the context provided in the switch label. If more
-complex text is needed in the switch than yes/no or on/off, use a **toggle**.
+## Introduction
 
-## Default
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/selectors-switch.png){:width="30%"}
+### When to use
+
+
+Use a switch if the user is turning something on or off, or saying yes or no to a question; there is no neutral option. Enabling preferences or settings is a prime use case for this component, explicitly stating the current status with text. 
+
+Use a toggle if more complex text is required. A switch **should not** be used for things like user agreements or if there are more than two options available.
```

Comment:
let's remove this section until we have the selector strip docs ready, then we can point at them.  What was previously "toggle" is now just a single-select selector strip with two options on it.

Reply:
Updated the sentence and commented out for now. Reads, `<!-- Use a single-select selector strip if more complex text is required. A switch **should not** be used for things like user agreements or if there are more than two options available.-->`